front-end/package.json

```diff
@@ -4,14 +4,32 @@
   "private": true,
   "proxy": "http://backend:5000",
   "dependencies": {
+    "bootstrap": "^3.3.7",
+    "lodash": "^4.17.11",
+    "moment": "^2.22.2",
```

Comment:
use `date_fns` instead. it interops with javascript native Date type, and also have simpler API